beacon-chain/sync/data_column_sidecars.go

```diff
@@ -107,39 +107,56 @@ func FetchDataColumnSidecars(
 		indicesToQueryMap, indicesStoredMap := categorizeIndices(params.Storage, root, indices)
 
 		if len(indicesToQueryMap) > 0 {
-			indicesByRootToQuery[root] = indicesToQueryMap
+			missingIndicesByRoot[root] = indicesToQueryMap
```

Comment:
not for this line but comment step 3a. 
> keep track of the sidecars that need to be queried
shouldn't this say custody columns needs to be queried?

Reply:
Here I assume that a sidecar is actually a data column sidecar.
This function does not assume anything about data column sidecars we should custody actually.
We can call this function with anything in the input, even if we do not need to custody the sidecars we fetch.